Rakefile

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Bosch Software Innovations GmbH 2016.
+# Part of the SW360 Portal Project.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+ROOT=File.dirname(__FILE__)
+
+TARGET_PREFIX=ENV['TARGET_PREFIX'] || "/opt/sw360/deploy"
+TARGET_NAME="sw360"
+TARGET_VERSION="1.4.0-SNAPSHOT"
+DEPENDENCIES_PACKAGE="sw360_dependencies"
+
+MAVEN_PARAMETERS=ENV['MAVEN_PARAMETERS'] || "-DskipTests"
+
+DEPLOY_DIR=File.expand_path(ENV['DEPLOY_DIR'] || File.join(ROOT, "deploy"))
+
+TMP_DIR_PREFIX="/tmp/sw360_package_tmp_dir_"
+TMP_DIR=TMP_DIR_PREFIX << `date +"%FT%Hh%Mm%Ss"`.strip
+
+DEV_CONTAINER_NAME = "sw360dev"
+DOCKERIZE=(ENV['DOCKERIZE'] || "true") == "true"
+
+print "########################################################################\n"
+print "# DEPLOY_DIR is set to: #{DEPLOY_DIR}\n"
+print "# MAVEN_PARAMETERS are set to: #{MAVEN_PARAMETERS}\n"
+print "# DOCKERIZE is set to: #{DOCKERIZE}\n"
+print "########################################################################\n\n"
+
+task :default => :install
```

Comment:
You are right, install is removed since maven is capable of doing this.

Maybe `package` could be a new default